projects/canopy/src/lib/side-nav/side-nav-bar/side-nav-bar.component.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../styles/mixins';
+
+.lg-side-nav-bar {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  padding: 0;
+  margin: var(--space-sm) 0 0 0;
+  font-size: var(--text-base-size);
+  font-weight: var(--tabs-list-item-font-weight);
+
+  > a {
+    text-decoration: none;
+  }
+
+  @include lg-breakpoint('md') {
+    margin: var(--space-sm) 0 0 0;
+  }
+
+  @include lg-breakpoint('lg') {
+    max-width: calc(33%);
```

Comment:
Does the `calc` do anything here? (I might be showing up a lack off css skills)

Reply:
That's actually a leftover and can be removed.